README.md

```diff
@@ -52,44 +52,20 @@ jupyter notebook
 ```
 Then open a notebook and select `nvt` from the `Kernel->Change Kernel` menu.
 
-#### Installing NVTabular with the Docker Container
+#### Installing NVTabular with Docker
 
-To use the NVTabular Docker Container, you'll first need to install the [NVIDIA Container Toolkit](https://github.com/NVIDIA/nvidia-docker) to provide GPU visibility to containers. NVTabular is available in the NVIDIA container repository at the following location: http://ngc.nvidia.com/catalog/containers/nvidia:nvtabular. You can pull the container by running the following command:
+Docker containers with NVTabular are available at the [NVIDIA Merlin container repository](https://ngc.nvidia.com/containers/merlin). There are four different containers, each providing different functionality:
```

Comment:
This link to container repo should be:

`https://ngc.nvidia.com/catalog/containers/nvidia:merlin`